scripts/cm_test_all_sandia

```diff
@@ -1142,6 +1142,7 @@ single_build_and_test() {
   then
     echo "        $MODULE_ENVIRONMENT" &>> reload_modules.sh
   fi
+  echo "        module purge" &>> reload_modules.sh
```

Comment:
@brian-kelley shouldn't this be placed before the `$MODULE_ENVIRONMENT` call? Otherwise I think this will blow away modules for some of the systems.

Reply:
@ndellingwood Are you concerned that``purge`` will undo commands in ``$MODULE_ENVIRONMENT``? I was worried about that too, but before my changes (before Apr 28) the script also did the machine-specific setup once and then purged before each build.

``sems-env`` is the first in the list of modules for the builds that need it, and I just made sure that ``module use`` isn't undone by purge.

Reply:
Thanks for checking, merge when you're ready